tools/python/test_custom_ops_pytorch_exporter.py

```diff
@@ -0,0 +1,153 @@
+import torch
```

Comment:
It is better to separate implementation and testing code in different folders
You could create a `tools/test` folder and move this test there.